documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_6_Usage_Control.md

```diff
@@ -1 +1,263 @@
+
+There are two ways to make usage restriction information available:
+
+1. Usage restriction policy information can be attached to the data that is about to be exchanged. This type of policy is called [sticky policy](https://www.computer.org/csdl/magazine/co/2011/09/mco2011090060/13rRUwI5Ubh). Following this approach, data is encrypted before it is sent to a Data Consumer, and it can only be decrypted if the Data Consumer fully and explicitly accepts the usage restrictions specified.
+
+2. A usage restriction policy can be stored independently of the data it relates to (for instance, in a central component, such as a PMP/PRP). In this case, the management component has the responsibility to exchange usage restriction information between different systems.
+
+The management of usage policies becomes especially important when data is to be exchanged across system boundaries. Every time data crosses system boundaries, the target system must be prepared for the protection of incoming data (i.e. it has to deploy the corresponding policy).
+
+#### **USAGE CONTROL IN A CONNECTOR**
+
+Usage Control only makes sense in an ecosystem where a certain level of trust can be established and maintained for all participants. To enable the establishment of trusted relationships, the central technological components used for data processing and data exchange need to be trustworthy. The IDS Connector is the central component for data exchange and data processing in the International Data Spaces, and thus a central component that needs to be trusted.
+
+The IDS Connector focuses on security and delivers a trusted platform, incorporating crucial building blocks:
+
+» identity & trust management for authenticating communicating parties (e.g., other Connectors) and shaping trusted relationships between partners;
+
+» a trusted platform as a baseline for secure data processing;
+
+» trustworthy communication based on authenticated and encrypted connections; and
+
+» Access & Usage Control.
+
+Instances of the Trusted Connector enable remote integrity verification, so the integrity of the deployed software stack can be guaranteed before granting access to data.
+
+The Trusted Connector guarantees a controlled execution environment for data services and supports the creation of trusted relationships. A general constraint is one that remains for all deployed IT systems: As long as physical or logical access is granted to administrators, protection against data theft by malicious partners is almost impossible to prevent. The International Data Spaces is seen as a network of partners that are provided with the technical means to fulfill their obligations and support in deciding what partners to trust and to define reasonable access conditions.
+
+#### **APACHE CAMEL INTERCEPTOR (EXAMPLE)**

```

Comment:
Do not use Apache Camle here, but saying "Message Router System inside the IDS Connector" or something similar.

Reply:
@juliapampus @HeinrichPet 
Removed Apache Camel and slightly changed the text